src/jdbc/cubrid/jdbc/jci/UConnection.java

```diff
@@ -1517,6 +1517,10 @@ public boolean getUseOldBooleanValue() {
 	public boolean getOracleStyleEmpltyString() {
 		return connectionProperties.getOracleStyleEmptyString();
 	}
+	
+	public int getClientCacheSize() { /* unit = MByte */
+		return connectionProperties.getClientCacheSize()*1024*1024;
```

Comment:
indentation for binary operator
```suggestion
		return connectionProperties.getClientCacheSize() * 1024 * 1024;
```